packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3213,6 +3213,26 @@ mixin ContainerRenderObjectMixin<ChildType extends RenderObject, ParentDataType
   }
 }
 
+/// Mixin for [RenderObject] that will call [markNeedsLayout] whenever the
+/// system fonts change
+///
+/// System fonts can change when OS install or remove a font. Use this mixin if
+/// the [RenderObject] uses [TextPainter] or text related painter to correctly
```

Comment:
What's a text related painter?

Reply:
using ui.Paragraph directly without going through text painter. I should update the doc comment